src/server/scripts/Events/love_in_air.cpp

```diff
@@ -52,9 +52,13 @@ class npc_love_in_air_supply_sentry : public CreatureScript
             {
                 lock = 0;
                 if (urand(0, 1))
-                    me->MonsterSay("Time is money, friend. Go go go!", LANG_UNIVERSAL, who->ToPlayer());
+                {
+                    me->AI()->Talk(0, who->ToPlayer());
```

Comment:
Use constants not magic numbers

Reply:
> Use constants not magic numbers

Ok, now I remove the magic numbers.
Thank you.